pkg/core/resources/apis/mesh/validators.go

```diff
@@ -449,6 +450,17 @@ func ValidateTargetRef(
 	return err
 }
 
+func ValidateMatch(match common_api.Match) validators.ValidationError {
```

Comment:
I think we also need to validate type since kubevalidator works only on k8s and not universal

Reply:
It is validated on open api schema